### PR TITLE
Update `package.json`

### PR DESCRIPTION
**Pull request by ruipenso**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -21,6 +21,11 @@
     "url": "https://github.com/seegno/angular-oauth2/issues"
   },
   "homepage": "https://github.com/seegno/angular-oauth2",
+  "dependencies": {
+    "angular": "^1.4.0",
+    "angular-cookies": "^1.4.0",
+    "query-string": "^1.0.0"
```
Comment (pgom) [inline package.json]: Any issues with bump this to `3.0.0`?

File: karma.conf.js [hunk 2]
```diff
@@ -14,9 +14,9 @@ module.exports = function(config) {
     basePath: './',
     browsers: [argv.browsers || 'Chrome'],
     files: [
-      'bower_components/angular/angular.js',
-      'bower_components/angular-cookies/angular-cookies.js',
-      'bower_components/query-string/query-string.js',
+      'node_modules/angular/angular.js',
+      'node_modules/angular-cookies/angular-cookies.js',
+      'node_modules/query-string/query-string.js',
```
Comment (pgom) [inline karma.conf.js]: If `query-string` is updated to `v3.0.0` the main file is `index.js`.

File: package.json (hunk 1 above)
Comment (ruipenso) [inline package.json]: Another PR. We need to evaluate BCs.

File: karma.conf.js (hunk 2 above)
Comment (ruipenso) [inline karma.conf.js]: Should we lock versions?

Comment (pgom) [inline karma.conf.js]: Yes.

Comment (ruipenso) [inline karma.conf.js]: Then, let's keep as is. We should update all dependencies in another PR.

**Pull request merged**